static/img/check-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="15" viewBox="0 0 18 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M2 7.71429L6.2 12L16 2" stroke="white" stroke-width="3" stroke-linecap="round"/>
+</svg>
```

Comment:
I think this one is unused?

Reply:
This is used as a CheckMark icon in src/components/CheckList/index.js

Reply:
But you've added two of the same icons:
1. static/img/check-icon.svg
2. src/img/check-icon.svg

One of them is a duplicate, please remove it! Thanks

Reply:
Removed in the commit 4090e6f